src/unitxt/loaders.py

```diff
@@ -140,7 +140,7 @@ class LoadFromIBMCloud(Loader):
     aws_secret_access_key_env: str
     bucket_name: str
     data_dir: str = None
-    data_files: Sequence[str]
+    data_files: Union[Sequence[str], Dict[str, str]]
```

Comment:
Can you please add a comment explaining what is the use of the dict here?
Also, i suggest adding a test with a dict value would to handle the coverage messages below.

Reply:
@matanor - Done, and also added another optional input type, is the current version ok?